README.md

```diff
@@ -22,11 +22,23 @@ Pre-build images are available on [DockerHub](https://hub.docker.com/r/existdb/e
 To download the image run:
 ```bash
 docker pull existdb/existdb:latest
-docker run -it -d -p 8080:8080 -p 8443:8443 existdb/existdb:latest
 ```
 
-You can now access eXist via [localhost:8080](localhost:8080) in your browser.
-Try to stick with matching internal and external port assignments, to avoid unnecessary reloads and connection issues.
+once the download is complete, you can run the image
+```bash
+docker run -it -d -p 8080:8080 -p 8443:8443 --name exist existdb/existdb:latest
```

Comment:
So that we are consistent, Can we use `-name existdb` please?

Reply:
i actually think this would add to the confusion, see joe question about what refers to what. `existdb` is the name of the image for all container using our stuff, `exist` is the name fo the running container in the readme, you are free to select another one, or docker will automatically select one for you.

Reply:
I disagree.